frontend/src/components/History/HistoryEditor/HistoryEditor.tsx

```diff
@@ -74,16 +75,22 @@ export default function HistoryEditor({
         <ModalHeader>{attempt.title}</ModalHeader>
         <ModalCloseButton />
         <ModalBody className={styles.body}>
-          <div className={styles.date}>{attempt.date}</div>
-          <div className={styles.bodyContainer}>
-            <div className={styles.description}>
-              {attempt.description.split("\n").map((desc, index) => (
-                <p key={index}>
-                  {desc}
-                  <br />
-                </p>
-              ))}
+            <div className={styles.bodyContainer}>
+              <div className={styles.detailsContainer}>
+                <div className={styles.date}>{attempt.date}</div>
+                  <div className={styles.description}>
+                    {attempt.description.split("\n").map((desc, index) => (
+                      <p key={index}>
+                        {desc}
+                        <br />
+                      </p>
+                    ))}
```

Comment:
is `attempt.description` the question description? if so, perhaps can reuse the `QuestionDescription.tsx` component which has the same logic of processing question description. this logic appeared multiple places, like question table and code room, so i refactored into a component to prevent duplication in one of the earlier PRs

Reply:
Ah, I didn't catch that previously, thanks!